setup.cfg

```diff
@@ -138,7 +116,7 @@ known_post_core=pandas.tseries,pandas.io,pandas.plotting
 sections=FUTURE,STDLIB,THIRDPARTY,PRE_CORE,DTYPES,FIRSTPARTY,POST_CORE,LOCALFOLDER
 
 known_first_party=pandas
-known_third_party=Cython,numpy,python-dateutil,pytz,pyarrow,pytest
```

Comment:
can you undo this change here, and send it in a different PR? I don't think this is related to the rest of the changes, is it?

Reply:
setup.cfg was rolled back completely and corrected.